app/views/posts/show.html.erb

```diff
@@ -6,7 +6,21 @@
   </head>
 
   <body>
-    <h1><%=params %></h1>
     <p> This is the Posts page and currently you have selected <%= params[:id] %> Post ID. </p>
+    
+        
+      <h2>Post: <%= @post.title %></h2> 
+      <span><%= @post.text %></span><br>
+      <span>Comments counter = <%= @post.comments_counter %></span>
+      <span>Likes counter = <%= @post.likes_counter %></span>
+      <br><br>
+      <% @post_comments.each do |comment| %>
+        <div>
+          <span><%= @user.name %></span> 
+          <span>: <%= comment.text %></span>
+        </div>
+      <% end %>
+      <br><br><br>
+  
   </body>
 </html>
```

Comment:
- [ ] Please fix the post-show page the @user is not presented in the controller the user will get this error if he tries to access the post-show page.
![image](https://user-images.githubusercontent.com/74744787/216771528-4e256175-f755-4b96-b7b7-7d20adadef52.png)
![image](https://user-images.githubusercontent.com/74744787/216771544-f70ab46a-8ce4-4df4-a219-cd36263636e3.png)
![image](https://user-images.githubusercontent.com/74744787/216771560-4539667c-27bd-40ca-a3b8-2ad6ca3961c0.png)